packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -373,6 +436,31 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
         }
     };
 
+    const menu_title = (
+        <>
+            <div>{localize('Menu')}</div>
```

Comment:
This should be a Text component.

Reply:
This title is wrapped with <Text/> inside <MobileDrawer/> . So there is just additional div and it seems to me, that everything is working correctly. could you please check as well